marketplace/src/components/BuyModal/BuyModalInfo.tsx

```diff
@@ -1,6 +1,5 @@
-import { FinalExecutionOutcome } from "@mintbase-js/auth";
 import { useMbWallet } from "@mintbase-js/react";
-import { TransactionSuccessEnum, buy, execute } from "@mintbase-js/sdk";
+import { FinalExecutionOutcome, TransactionSuccessEnum, buy, execute } from "@mintbase-js/sdk";
```

Comment:
I guess this was moved from auth to sdk? Is there a corresponding PR for that?

Reply:
funny enough, we have that type in the two packages.